pkg/composer/format.go

```diff
@@ -41,19 +40,19 @@ func PronounceInt(d int) string {
 	case 2:
```

Comment:
I think this entire switch and the panic below it can be replaced with one call to `strconv.Itoa()` now

Reply:
Similarly the "zero " in PronounceBearing can be replaced with "0 "

Reply:
updated